<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessengerClass</class>
 <widget class="QMainWindow" name="MessengerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messenger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">/*-----QWidget-----*/
QWidget
{
	background-color: #0f0f0f;
	color: #fff;
	border-color: #000000;

}


/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #fff;
	font-weight: bold;
	border-color: #000000;

}


/*-----QPushButton-----*/
QPushButton
{
	background-color: #d10000;
	color: #fff;
	font-weight: bold;
	border: 1px solid #d10000;  
	border-radius: 2px;
	padding: 5px;

}


QPushButton::hover
{
	background-color: #ad0000;
	color: #fff;

}


QPushButton::pressed
{
	background-color: #e00000;
	color: #fff;

}


/*-----QToolButton-----*/
QToolButton
{
	background-color: transparent;
	color: #000000;
	border-style: solid;
	border-color: #000000;

}


QToolButton::hover
{
	background-color: #d10000;
	color: #000000;
	border-radius: 15px;
	border-color: #d10000;

}


QToolButton::pressed
{
	background-color: #d10000;
	color: #000000;
	border-style: solid;
	border-width: 0px;

}


/*-----QLineEdit-----*/
QLineEdit{
	background-color: #4d4d4d;
	color: #fff;
	font-weight: bold;
	border-style: solid;
	border-radius: 5px;
	padding: 5px;

}


/*-----QCheckBox-----*/
QCheckBox
{
	background-color: transparent;
	color: #b9b9bb;
	font-weight: bold;

}


QCheckBox::indicator
{
    color: #b1b1b1;
    background-color: #00111d;
    border: 1px solid #d10000;
    width: 12px;
    height: 12px;

}


QCheckBox::indicator:checked
{
    image:url(&quot;./ressources/check.png&quot;); /*To replace*/
	background-color: #d10000;
    border: 1px solid #d10000;

}


QCheckBox::indicator:unchecked:hover
{
    border: 1px solid #ff0000;

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}
</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>520</width>
      <height>246</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel">
       <property name="text">
        <string>My Login:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lbl_my_login">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QListWidget" name="list_friends"/>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLineEdit" name="le_message"/>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btn_send">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>list_friends</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MessengerClass</receiver>
   <slot>user_selected(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_send</sender>
   <signal>clicked()</signal>
   <receiver>MessengerClass</receiver>
   <slot>send_message()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>user_selected(QListWidgetItem*)</slot>
  <slot>send_message()</slot>
 </slots>
</ui>
